src/man/sssd-ad.5.xml

```diff
@@ -1356,6 +1356,71 @@ ad_gpo_map_deny = +my_pam_service
                     </listitem>
                 </varlistentry>
 
+                <varlistentry>
+                    <term>dyndns_dot (boolean)</term>
+                    <listitem>
+                        <para>
+                            This option enables DNS-over-TLS protocol for DNS
+                            updates. The nsupdate utility must support DoT -
+                            check the <emphasis>man nsupdate</emphasis> before
+                            enabling it in SSSD.
+                        </para>
+                        <para>
+                            Default: False (use the traditional DNS protocol)
+                        </para>
+                    </listitem>
+                </varlistentry>
+
+                <varlistentry>
+                    <term>dyndns_dot_cacert (string)</term>
```

Comment:
Will it be possible to specify a directory? OpenSSL in CentOS Stream 10 and Fedora 41+ is switching to default not using a single file CA bundle due to performance issues, so we should expect people wanting to use a directory of certs as well for custom bundles.

Reply:
No, I don't see any option in nsupdate. https://bind9.readthedocs.io/en/v9.20.1/manpages.html#id126

Reply:
Could you open a ticket at https://gitlab.isc.org/isc-projects/bind9? You have more insights about it than I do.